File: app/src/main/java/me/tagavari/airmessage/util/NotificationSummaryMessage.java
package me.tagavari.airmessage.util;

//Represents an item in a summary notification
public class NotificationSummaryMessage {
	private final String conversation;
	private final String body;
	
	public NotificationSummaryMessage(String conversation, String body) {
		this.conversation = conversation;
		this.body = body;
	}
	
	public String getConversation() {
		return conversation;
	}
	
	public String getBody() {
		return body;
	}
}

File: common/me/tagavari/airmessage/common/util/NotificationSummaryMessage.kt
package me.tagavari.airmessage.common.util

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

//Represents an item in a summary notification
@Parcelize
data class NotificationSummaryMessage(val conversationID: Int, val title: String, val description: String) : Parcelable